pkg/apis/v1/labels.go

```diff
@@ -47,6 +47,7 @@ const (
 	ManagedByAnnotationKey             = Group + "/managed-by"
 	NodePoolHashAnnotationKey          = Group + "/nodepool-hash"
 	NodePoolHashVersionAnnotationKey   = Group + "/nodepool-hash-version"
+	V1Beta1KubeletConfiguration        = Group + "/v1beta1-kubelet"
```

Comment:
Should this label be similar to the compatibility annotation that we were using for a while?

Reply:
Will be added in a separate PR